engine/minskyTensorOps.cc

```diff
@@ -191,6 +195,7 @@ namespace minsky
     registerOps<TensorBinOp, OperationType::log, OperationType::copy>(*this);
     registerOps<MinskyTensorOp, OperationType::copy, OperationType::sum>(*this);
     registerOps<GeneralTensorOp, OperationType::sum, OperationType::numOps>(*this);
+    registerOps<RavelTensor, OperationType::sum, OperationType::numOps>(*this);
```

Comment:
It is a single type, so you should do registerType<RavelTensor>(OperationType::ravel);

Note the presence of the static variable tensorOpFactor on the TimeOp is a bit of a mistake (harmless one), but I've removed in a subsequent commit.